### PR TITLE
Fixed a bunch of compiler warnings

### PR DESCRIPTION
**Pull request by weyert**

### CODE REVIEW

File: Classes/Views/UIColor+Hex.swift [hunk 1]
```diff
@@ -18,7 +18,7 @@ extension UIColor {
             cString.remove(at: cString.startIndex)
         }
 
-        if ((cString.characters.count) != 6) {
+        if ((cString.count) != 6) {
```
Comment (Sherlouk) [inline Classes/Views/UIColor+Hex.swift]: `if cString.count != 6 {`?

Comment (BasThomas) [reply Classes/Views/UIColor+Hex.swift]: Can clean up all of them here I suppose. Think Ryan coped this straight from SO 😜

Comment (Sherlouk) [reply Classes/Views/UIColor+Hex.swift]: Noo Ryan wouldn't do that... would he?? 😉

Comment (BasThomas) [reply Classes/Views/UIColor+Hex.swift]: So: omit `:String` in line 15
Omit `()` in 17
Omit `(())` in 21
Omit `UIColor` in 22.

**Comment by weyert** [comment]
Updated!

**Pull request merged**